src/scss/_settings.variables.colors.scss

```diff
@@ -1,4 +1,9 @@
 $blue: #15CBF4;
+<<<<<<< HEAD
```

Comment:
Looks like this was missed on a rebase, last thing 🤞

Reply:
Oh my goodness!!! 🙈